plugins/node.d/snmp__if_

```diff
@@ -155,11 +155,12 @@ if ($ARGV[0] and $ARGV[0] eq "config") {
 
     print "host_name $host\n" unless $host eq 'localhost';
 
-    my $alias = $session->get_single($ifEntryAlias) ||
+    my $ENV{alias} ||
```

Comment:
Maybe you omitted `$alias = ` here?

Reply:
Dang it, you're absolutely right - I'll fix that up.